db/schema.rb

```diff
@@ -175,6 +160,21 @@
 
   add_index "roles", ["name"], name: "roles_name_idx", unique: true, using: :btree
 
+  create_table "steps", force: :cascade do |t|
+    t.integer  "user_id"
+    t.string   "status",              limit: 255
+    t.datetime "created_at"
+    t.datetime "updated_at"
+    t.integer  "position"
+    t.integer  "proposal_id"
+    t.datetime "approved_at"
```

Comment:
does it make sense to rename this column to `stepped_at` to align the naming logic?